docs/docs/tutorials/run-notebooks-as-a-job.md

```diff
@@ -64,7 +64,24 @@ As mentioned above, the notebook job will run as an Argo-Workflows workflow. Thi
 :::
 
 :::note
-Only users in the `admin` or `developer` groups will have access to create notebook jobs. For more information regarding users and groups, please visit [How to configure Keycloak](/how-tos/configure-keycloak-howto.md)
+Only users in the `admin` or `developer` groups will have access to create notebook jobs. For more information regarding users and groups, please visit [How to configure Keycloak](/how-tos/configure-keycloak-howto.md).
+
+If your user is not part of one of the mentioned groups, you might see an error
+like this one when clicking on the job details:
+
+`Server returned status code 403 with message: 'workflows.argoproj.io is forbidden: User "system:serviceaccount:dev:argo-viewer" cannot create resource "workflows" in API group "argoproj.io" in the namespace "dev"`
```

Comment:
Note: I use single backticks here on purpose: it allows showing the entire message inline and wraps it around to content width while still formatting it as monospace. Using three backticks would either make it too long (if one line) or would require breaking it into multiple lines (but then it would be harder for users to find this message if they search the docs by error message).

Reply:
@nkaretnikov you mentioned [here](https://github.com/nebari-dev/nebari/issues/2444#issuecomment-2132711794) that `developer` was not required. Seems like we need to update the wording on the docs here?